apps/hmc/hmc-cft-hearing-service/perftest.yaml

```diff
@@ -6,10 +6,14 @@ spec:
   releaseName: hmc-cft-hearing-service
   values:
     java:
-      memoryLimits: "3Gi"
-      memoryRequests: "2Gi"
-      cpuLimits: "2000m"
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: "2048Mi"
```

Comment:
Based on current memory utilisation in Production (currently averaging around 1.7gb, perhaps because the pods have been up for 3 weeks), when you enable the HPA, it'll want to scale up to 3 pods per cluster. Based on the algorithm, it won't scale back to 2 until there's a restart. Even increasing the threshold to 120% won't be sufficient, so I'd suggest either slightly increasing the memoryRequests, the threshold or both. For example, the following configuration might be suitable: memoryRequests = 2304Mi (=2.25Gb); threshold = 120%. This will provide sufficient headroom for scaling in both directions, whilst ensuring it can scale before hitting the limits (2304*1.2 < 3072).